src/download.jl

```diff
@@ -0,0 +1,72 @@
+using Printf
+using DataDeps
+
+# generate_dataset_url helper function
+# This function takes in a dataset code of format "GADM/<country_code>"
+# It returns the URL of the gpkg dataset of the country
+
+function generate_dataset_url(dataset_code)
```

Comment:
I would rename the function to simply `dataurl`. We try to avoid `_` in names when we can in Julia to make the code less cryptic.